tools/cpp/windows_cc_configure.bzl

```diff
@@ -725,11 +746,12 @@ def configure_windows_toolchain(repository_ctx):
     )
 
     template_vars = dict()
-    msvc_vars = _get_msvc_vars(repository_ctx, paths)
-    template_vars.update(msvc_vars)
-    template_vars.update(_get_clang_cl_vars(repository_ctx, paths, msvc_vars))
+    msvc_vars_x64 = _get_msvc_vars(repository_ctx, paths, "x64")
+    template_vars.update(msvc_vars_x64)
+    template_vars.update(_get_clang_cl_vars(repository_ctx, paths, msvc_vars_x64))
     template_vars.update(_get_msys_mingw_vars(repository_ctx))
-
+    template_vars.update(_get_msvc_vars(repository_ctx, paths, "x86"))
```

Comment:
@mai12 Have you noticed a significant longer time when first running Bazel on Windows. It's because we have to run the vcvarsall.bat script twice now, which is quite slow. But from https://github.com/meteorcloudy/my_tests/commit/b6d3bc43b96b635d9045e18d7efcaca048bbf6dd#diff-be9303cb6d3994a5092fee62598e3df8, you can see, the `INCLUDE` env var doesn't change, `PATH` env var only has one more x86 directory at the beginning, `LIB` changes x64 to x86. Since they are quite predictable, can we avoid running the vcvarsall.bat twice to save some time for our users?